rest_framework_swagger/docgenerator.py

```diff
@@ -20,6 +20,9 @@ class DocumentationGenerator(object):
     # Response classes defined in docstrings
     explicit_response_types = dict()
 
+    def __init__(self, for_user):
```

Comment:
That's also true. 

I think there's some benefit to self-documenting the default argument value and the cost of one AnonymousUser instance is also trivial. On the other hand, aesthetically I don't like the constructor call in the middle of the signature. I think I will switch to `None`.